runtime-modules/content/src/benchmarks/mod.rs

```diff
@@ -524,3 +618,212 @@ where
         expected_channel_state_bloat_bond,
     }
 }
+
+fn assert_last_event<T: Config>(generic_event: <T as Config>::Event) {
+    let events = System::<T>::events();
+    let system_event: <T as frame_system::Config>::Event = generic_event.into();
+    // compare to the last event record
+    let EventRecord { event, .. } = &events[events.len() - 1];
+    assert_eq!(event, &system_event);
+}
+
+fn worst_case_content_moderation_actions_set() -> BTreeSet<ContentModerationAction> {
+    CONTENT_MODERATION_ACTIONS.iter().cloned().collect()
+}
+
+fn worst_case_channel_agent_permissions() -> ChannelAgentPermissions {
+    CHANNEL_AGENT_PERMISSIONS.iter().cloned().collect()
+}
+
+fn worst_case_scenario_collaborators<T: RuntimeConfig>(
+    start_id: u32,
+    num: u32,
+) -> BTreeMap<T::MemberId, ChannelAgentPermissions>
+where
+    T::AccountId: CreateAccountId,
+{
```

Comment:
maybe add `assert!(start_id + num <= MAX_COLABORATOR_IDS)` to avoid benchmarks breaking due to future changes?

Reply:
Done in https://github.com/Joystream/joystream/pull/4067/commits/d37ae86ffc0fa1b75e80a998c9907e224466a719